lms/static/js/spec/main.js

```diff
@@ -95,6 +95,7 @@
             'js/bookmarks/views/bookmarks_list': 'js/bookmarks/views/bookmarks_list',
             'js/bookmarks/views/bookmark_button': 'js/bookmarks/views/bookmark_button',
             'js/views/message_banner': 'js/views/message_banner',
+            'js/commerce/views/receipt_view': 'js/commerce/views/receipt_view',
```

Comment:
I would move this up to the section called ``// Manually specify LMS files that are not converted to requireJS``. As a nit, I also like to maintain these files in alphabetical order so that it is easier to find if the file is present or not.